app/services/applications/change_funded_place.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Applications
+  class ChangeFundedPlace
+    include ActiveModel::Model
+    include ActiveModel::Attributes
+
+    attribute :application
+    attribute :funded_place, :boolean
+
+    validates :application, presence: { message: I18n.t("application.missing_application") }
+    validates :funded_place,
+              inclusion: {
+                in: [true, false],
+                message: I18n.t("application.missing_funded_place"),
+              }
+    validate :accepted_application
+    validate :eligible_for_funding
+    validate :cohort_has_funding_cap
+
+    def change
+      return false unless valid?
+
+      application.update!(funded_place:)
+    end
+
+  private
+
+    delegate :cohort, to: :application
+
+    def accepted_application
+      return if application.accepted?
+
+      errors.add(:application, I18n.t("application.cannot_change_funded_status_from_non_accepted"))
+    end
+
+    def eligible_for_funding
```

Comment:
I think we have an extra check added on the other pr if you don't mind adding it here please https://github.com/DFE-Digital/early-careers-framework/pull/4738/files#diff-9312911905848cf62a214e3cbc8cfd98f3308145ff4a9b6123d8cfe1d5e0cbc9R13